adcm_client/objects.py

```diff
@@ -729,6 +740,15 @@ class Task(BaseAPIObject):
     selector = None
     status = None
     url = None
+    object_type = None
+
+    @min_server_version('2020.08.27.00')
+    def action(self) -> "Action":
+        try:
+            return TASK_PARENT[self.object_type](
+                self._api, id=self.selector[self.object_type]).action(id=self.action_id)
+        except KeyError:
+            raise DoesNotExistAction from None
```

Comment:
I don't like this name. It is better to call it UnknownTaskParent or something like this.